src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -587,24 +566,19 @@ - (BOOL)setOperationalCertificateIssuer:(nullable id<MTROperationalCertificateIs
         return NO;
     }
 
-    VerifyOrReturnValue([self checkIsRunning], NO);
-
-    __block BOOL success = NO;
-    dispatch_sync(_chipWorkQueue, ^{
-        VerifyOrReturn([self checkIsRunning]);
-
+    auto block = ^() {
```

Comment:
So why are some of the blocks `^BOOL()` and this one is just `^()`?

Reply:
That was a mistake. In practice it does not matter since the type is inferred from what is returned in the block. To make it simple I replaced all `^{}` and `^BOOL()` which keeps some explicit behavior at compile time since those blocks are passed to some methods that actually expects no return value or a bool value.